test/ctia/bundle/routes_test.clj

```diff
@@ -1002,4 +1009,4 @@
          (is (= 201 (:status incident-post-res)))
          (is (= 201 (:status link-res)))
          (is (= 200 (:status bundle-get-res)))
-         (is seq (-> bundle-get-res :parsed-body :casebooks)))))))
+         (is (seq (-> bundle-get-res :parsed-body :casebooks))))))))
```

Comment:
:( thanks for this one

I checked that there are no more occurences: `rg --pcre2 '\(is\s(?!\().{1}'`